src/ActionModal/components/ActionModal.vue

```diff
@@ -698,11 +700,23 @@ export default {
         let transactionData
         // Polkadot loads transaction data automatic
         if (this.network.network_type === "cosmos") {
+          const coinLookup = this.currentNetwork.coinLookup.find(
+            ({ viewDenom }) =>
+              viewDenom === this.networkFees.transactionFee.denom
+          )
           transactionData = await this.transactionManager.getCosmosTransactionData(
             {
               memo,
               gasEstimate: this.networkFees.gasEstimate,
-              fee: [this.networkFees.transactionFee],
+              // convert fee to chain values
+              fee: [
+                {
+                  amount: BigNumber(this.networkFees.transactionFee.amount)
```

Comment:
move the conversion into `getCosmosTransactionData`

Reply:
Done